graspologic/match/qap.py

```diff
@@ -427,6 +427,7 @@ def _quadratic_assignment_faq(
     else:
         P0 = np.atleast_2d(P0)
         _check_init_input(P0, n_unseed)
+        P0 = P0[perm_A][:, perm_B]
```

Comment:
@asaadeldin11 is this the permutation that makes sense here?

Reply:
yeah i think so. Best way to test it out would be pass in an instance with known optimum, with PO = solution perm and shuffle_input = True. Then check `nit` (expect = 1) and `perm_inds_` and `score_` are what we expect?

Reply:
@asaadeldin11 do you expect the gradient from this initial position to be the 0 matrix?

Reply:
(from the optimal permutation I mean)

Reply:
@asaadeldin11 see the test I added

Reply:
saw the comment in the test, good point